server/store/datastore/migration/common.go

```diff
@@ -170,7 +170,7 @@ func alterColumnNull(sess *xorm.Session, table, column string, null bool) error
 	dialect := sess.Engine().Dialect().URI().DBType
 	switch dialect {
 	case schemas.MYSQL:
-		sql := fmt.Sprintf("SHOW COLUMNS FROM `%s` WHERE lower(field) = '%s'", table, strings.ToLower(column))
+		sql := fmt.Sprintf("SHOW COLUMNS FROM `%s` WHERE lower(field) = '%s'", table, column)
```

Comment:
Would skip this in this PR to avoid accidentally side effects and put it into a separate one.